.flake8

```diff
@@ -0,0 +1,19 @@
+[flake8]
+max-line-length = 120
+extend-ignore =
+    # Whitespace before ':'
+    E203,
+    # imported but unused
+    F401,
+    # do not use bare 'except'
+    E722
```

Comment:
Hm, but may be we should keep this as an error?

Reply:
Yes, eventually. But now it caught an error in a file that I did not code and I did not want to introduce a bug so thought it best to wait until the team is back for this change.

Reply:
I have now enabled the rule and excluded the single line to be revisited later.